### ISSUE TITLE
Automatic mode not working in Icecat

### ISSUE THREAD

**Issue by Perflyst** [stoically/temporary-containers#307]
- Temporary Containers Version: 
- Firefox Version: 60.7.0esr (64-bit)
- Operating System + Version: Ubuntu 18.04.3 LTS
- Non-default Options/Preferences in Temporary Containers: Automatic mode = true
- Other installed Add-ons + Version: None

### Actual behavior
Automatic mode (new tab = new container) not working

### Expected behavior
New tab = create new container

### Steps to reproduce
1. Install Icecat 
2. Install TC
3. Enable Automatic mode
4. See the bug

### Notes
Debug log from opening a new tab with automatic mode enabled:

<details>
<br>

```
Fri, 27 Sep 2019 18:58:50 GMT "[onUpdated] tab updated" {"id":47,"index":22,"windowId":3,"highlighted":false,"active":false,"pinned":false,"status":"complete","hidden":false,"discarded":false,"incognito":false,"width":1920,"height":947,"lastAccessed":1569610730582,"audible":false,"mutedInfo":{"muted":false},"isArticle":false,"isInReaderMode":false,"sharingState":{"camera":false,"microphone":false},"cookieStoreId":"icecat-default","url":"about:newtab","title":"New Tab"} {}  log.js:39:28
------------------------------------------  log.js:40:7
Fri, 27 Sep 2019 18:58:50 GMT "[onUpdated] url didnt change, not relevant" 47 {}  log.js:39:28
------------------------------------------  log.js:40:7
Fri, 27 Sep 2019 18:58:50 GMT "[onCreated] tab created" {"id":47,"index":22,"windowId":3,"highlighted":true,"active":true,"pinned":false,"status":"complete","hidden":false,"discarded":false,"incognito":false,"width":1920,"height":947,"lastAccessed":1569610730629,"audible":false,"mutedInfo":{"muted":false},"isArticle":false,"isInReaderMode":false,"sharingState":{"camera":false,"microphone":false},"cookieStoreId":"icecat-default","url":"about:newtab","title":"New Tab"}  log.js:39:28
------------------------------------------  log.js:40:7
Fri, 27 Sep 2019 18:58:50 GMT "[maybeReloadInTempContainer] not a home/new/moz tab or disabled, we dont handle that" {"id":47,"index":22,"windowId":3,"highlighted":true,"active":true,"pinned":false,"status":"complete","hidden":false,"discarded":false,"incognito":false,"width":1920,"height":947,"lastAccessed":1569610730629,"audible":false,"mutedInfo":{"muted":false},"isArticle":false,"isInReaderMode":false,"sharingState":{"camera":false,"microphone":false},"cookieStoreId":"icecat-default","url":"about:newtab","title":"New Tab"}  log.js:39:28
------------------------------------------  log.js:40:7
Fri, 27 Sep 2019 18:58:50 GMT "[onActivated]" {"tabId":47,"windowId":3}  log.js:39:28
------------------------------------------  log.js:40:7
Error: Invalid contextual identity: icecat-default  ext-contextualIdentities.js
```
</details>

**Comment by stoically** [comment]
https://www.bountysource.com/issues/81034665-automatic-mode-not-working-in-icecat

**Comment by stoically** [comment]
Fixed in [v1.4beta1](https://github.com/stoically/temporary-containers/releases).
